tools/REALTEK_RTL8195AM.py

```diff
@@ -0,0 +1,161 @@
+"""
```

Comment:
@theotherjimmy I asked Realtek guys to get rid of this file from tools directory, if more and more targets start keeping their scripts directly there it'll become a mess. My proposed solution was to create a subdirectory called 'tools/targets' and move this file there to keep 'tools' clean and easier to read/navigate. What do you think?